### ISSUE TITLE
Extra lines included in doc comments

### ISSUE THREAD

**Issue by lnicola** [rust-analyzer/rust-analyzer#4040]
`serde` has this doc comment on its `Serialize` trait:

```rust
////////////////////////////////////////////////////////////////////////////////

/// A **data structure** that can be serialized into any data format supported
/// by Serde.
///
```

docs.rs doesn't render the `////////////////////////////////////////////////////////////////////////////////` line, but `rust-analyzer` does:

![image](https://user-images.githubusercontent.com/308347/79684781-ae2c2a00-823c-11ea-8cfe-fe8480fbd4ef.png)

**Comment by Aloso** [comment]
According to the [reference](https://doc.rust-lang.org/reference/comments.html), lines starting with more than 3 forward slashes aren't doc comments:

OUTER_LINE_DOC :
   `///` (~`/` ~[`\n` _IsolatedCR_]*)?

**Comment by Aloso** [comment]
According to the [reference](https://doc.rust-lang.org/reference/comments.html), lines starting with more than 3 forward slashes aren't doc comments:

OUTER_LINE_DOC :
   `///` (~`/` ~[`\n` _IsolatedCR_]*)?

**Comment by Aloso** [comment]
According to the [reference](https://doc.rust-lang.org/reference/comments.html), lines starting with more than 3 forward slashes aren't doc comments:

OUTER_LINE_DOC :
   `///` (~`/` ~[`\n` _IsolatedCR_]*)?

**Comment by lnicola** [comment]
Fixed in #4184.
